services/src/messaging/src/main/java/org/openkilda/messaging/command/reroute/RerouteFlows.java

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command.reroute;
+
+import org.openkilda.messaging.command.CommandData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+
+@EqualsAndHashCode(callSuper = false)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RerouteFlows extends CommandData {
+
+    @JsonProperty("reason")
+    private String reason;
+
+    @JsonCreator
+    public RerouteFlows(@NonNull @JsonProperty("reason") String reason) {
+        this.reason = reason;
+    }
+
+    public String getReason() {
```

Comment:
Add `@Data` or `@Value` to the class definition.

PS Which flows should be removed by this command?

Reply:
This class was written to avoid code repetition. RerouteAffectedFlows and RerouteInactiveFlows are extends from it.